ISISComputingGroup/EPICS-StreamDevice
# Ticket 2916: Reduce continuous logging when mismatched or disconnected

See https://github.com/ISISComputingGroup/IBEX/issues/2916

Other PRs:
https://github.com/ISISComputingGroup/EPICS-DeviceEmulator/pull/52
https://github.com/ISISComputingGroup/EPICS-IOC_Test_Framework/pull/63
https://github.com/ISISComputingGroup/EPICS-ioc/pull/242

To test:
* Rebuild StreamDevice and CCD100 support/ioc
* Run the CCD100 pointing at the emulator

Confirm that (per PV):
- [ ] When the IOC disconnects only one messages is logged
- [ ] When the IOC reconnects a message is logged
- [ ] When there is a mismatch only 3 messages are logged
- [ ] When the mismatch changes to a different string another 3 messages are logged
- [ ] When the match is successful again a message is logged

Includes commit from https://github.com/paulscherrerinstitute/StreamDevice/commit/ca01ba7c35a36c02206e66de7c4429ca8ed42550

Once merged to our repo, contact PSI to see if they would like to merge.
